tests/testthat/setup.R

```diff
@@ -0,0 +1,27 @@
+jdbcDriverFolder <- tempfile("jdbcDrivers")
+DatabaseConnector::downloadJdbcDrivers("postgresql", pathToDriver = jdbcDriverFolder)
+connectionDetails <- DatabaseConnector::createConnectionDetails(
+    dbms = "postgresql",
+    user = Sys.getenv("CDM5_POSTGRESQL_USER"),
+    password = URLdecode(Sys.getenv("CDM5_POSTGRESQL_PASSWORD")),
+    server = Sys.getenv("CDM5_POSTGRESQL_SERVER"),
+    pathToDriver = jdbcDriverFolder)
+
+cdmDatabaseSchema <- Sys.getenv("CDM5_POSTGRESQL_CDM_SCHEMA")
+vocabularyDatabaseSchema <- Sys.getenv("CDM5_POSTGRESQL_CDM_SCHEMA")
+cohortDiagnosticsSchema <- Sys.getenv("CDM5_POSTGRESQL_COHORT_DIAGNOSTICS_SCHEMA")
+tempEmulationSchema <- NULL
+# Temp Random Schema to prevent collisions/issues when multiple test instances run
+cohortDatabaseSchema <- paste0("cohort_diagnostics_test", stringi::stri_rand_strings(1,5))
```

Comment:
maybe better to make this dateTime() ? that would ensure less likely to have a clash